base/src/org/compiere/acct/Doc.java

```diff
@@ -2505,19 +2505,18 @@ public Boolean isReverseWithOriginalAccounting()
 	 * Generate Reverse using Orginal Accounting
 	 * @return
 	 */
-	private String generateReverseWithOriginalAccounting() {
-		getReversalFactAcct().stream().forEach(factAcct -> {
+	private String generateReverseWithOriginalAccounting(MAcctSchema asch) {

```

Comment:
@carlosaparadam can please use complete variable description is best to developer a good DSL in code.

generateReverseWithOriginalAccounting(MAcctSchema originalAccountSchema) 

Also adding the comment parameter in documentation